src/validation.cpp

```diff
@@ -1088,8 +1107,8 @@ static bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state,
                 tx.GetTotalSize(),
                 nConflictingFees
             );
-            if (plTxnReplaced)
```

Comment:
Why this condition removed?

Reply:
Earlier it was a pointer and optional. So the check ensured that null pointer is not dereferenced. Now it is a vector inside 'opt' which is passed around as a reference. So we can safely access its contents as they are always initialized.